docs/data/charts/export/PrintChart.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
+import {
+  ScatterChartPro,
+  ScatterChartProApi,
+} from '@mui/x-charts-pro/ScatterChartPro';
+import { data } from './randomData';
+
+export default function PrintChart() {
+  const apiRef = React.useRef<ScatterChartProApi>(undefined);
```

Comment:
This seems better than the grid's `useGridRefApi` because it's a simple ref. What do you think?